### ISSUE TITLE
Feature Request: Warlord Traits.

### ISSUE THREAD

**Issue by MandaPii** [BSData/wh40k#1838]
**File/Catalogue:** 
All
**BattleScribe version:** 2.01.
2.01.08 (Android)
**Platform:** Windows/iOS/Android/Mac/Linux
Windows/Android
**Dropbox:** Yes/No
Yes
**Description:** Detailed issue description.
I'd like to request the selection of the warlord and warlord trait. I would like this to be added to the armylist so I can easily view this. Me and my friend keep forgetting about warlord traits as we use Battlescribe sheets during battle instead of browsing the codex or index for everything we don't remember.

**Comment by WindstormSCR** [comment]
The problem with this is that _any_ character can be the warlord, so to implement this would involve adding minimum seven lines to every single character (without the check logic) which would significantly bloat all the catalog files. 

A better option (That I use myself) is to use an index card to track warlord trait, relics, and orders/powers per game. Then you'll have a physical reminder.

**Comment by GenWilhelm** [comment]
All it takes is a single shared entry group that's linked to each character. I hardly think that would cause any significant bloat.

And I know for many people, it's easier to have it on the roster who was the warlord, and which trait they had in mind when they built the list. It's not a big deal for Index armies, since they all have access to the same 3 traits, but for Codex armies where you have 6 generic traits, and 5-7 chapter-esque traits, it is easier to know what your options are.

Ultimately, it's down to each catalogue maintainer whether or not they want to include warlord traits, as it's not integral to list building. Grey Knights, AdMech, and Tyranids have them, but as I said in the previous thread (#1238), it would be a larger undertaking for the bigger factions.

**Comment by WindstormSCR** [comment]
Fair point. Up to the CAT maintainer then, based on the number of characters affected and level of existing bloat.

**Comment by Mikle103** [comment]
How much of a bloat would it be if it was just an addition list like the regimental one for AM
eg below
  
![untitled](https://user-images.githubusercontent.com/30399027/33408523-dd89c62e-d5c6-11e7-868f-b04b78360422.jpg)

**Comment by alphalas** [comment]
Then how do you indicate which model has it? To have it properly indicate on the model means adding a link to a SEG for every character model in the roster, and adding a boatload of conditionals. It’s not the links that’ll cause bloat, it’s the conditions

**Comment by Mikle103** [comment]
Sorry I am not sure of how all the inter connections work or the conditions. I was just trying to suggest a easier way of doing it.
As for indicating which character is the warlord (shoot me down if wrong) could you have a tick box saying warlord (one per army)???
Again don't know the complexity of the background work needed.

**Comment by alphalas** [comment]
Ehhh... I’ll look into it and see what happens.

**Comment by radionausea** [comment]
The Adeptus Mechanicus catalogue already had the Warlord trait as selectable.

**Comment by CrusherJoe** [comment]
I updated the checklist with the ones I know either I or @alphalas had done.

**Comment by FarseerVeraenthis** [comment]
Ticked a couple more off :smile:

**Comment by toadchild** [comment]
I see that Imperium - Space Marines is checked off above but I cannot see how to set a warlord using v41 of that catalogue.

**Comment by DarkTrainer** [comment]
It would be nice if a 'single' warlord trait could be selected from the Necron list, rather than 'all'. Even if it was more like a wargear selection. Highlight it, select the one trait you want, then it would only print one rather than a huge page.

![image](https://user-images.githubusercontent.com/49451797/55821614-c0647b80-5ab2-11e9-9b54-c2d6e0b43fce.png)
